www-servers/lighttpd/files/conf/lighttpd.conf

```diff
@@ -17,16 +17,15 @@ server.modules = (
 #    "mod_redirect",
```

Comment:
You didn't do the copy-and-reference-new-one-in-ebuild thing?

Reply:
I must admit that I haven't set up a whole VM for every build system to which I try to contribute.  What are the underlying commands that the ebuild command magically does for me?

Reply:
See https://github.com/gentoo/gentoo/pull/23900#discussion_r789901391 but s/.initd/.conf/ :)

(Let me know if more help is needed there.)

Reply:
I think that I see what you mean.  If that is what you want me to do, I'll try it, but might need some help.

Reply:
Is this what you're looking for?  (along with the new file, of course)
```
--- a/www-servers/lighttpd/lighttpd-1.4.64-r1.ebuild
+++ b/www-servers/lighttpd/lighttpd-1.4.64-r1.ebuild
@@ -172,7 +172,7 @@ src_install() {
 
        # configs
        insinto /etc/lighttpd
-       doins "${FILESDIR}"/conf/lighttpd.conf
+       newins "${FILESDIR}"/conf/lighttpd.conf-r1 lighttpd.conf
        doins "${FILESDIR}"/conf/mime-types.conf
        doins "${FILESDIR}"/conf/mod_cgi.conf
        doins "${FILESDIR}"/conf/mod_fastcgi.conf
```

Reply:
Heh, that was harder than I expected it to be (`doins` vs `newins`), yes exactly!

(With obviously a `cp` for lighttpd.conf -> lighttpd.conf-r1 for the ebuild to pick up on)

Reply:
change pushed